package.json

```diff
@@ -294,7 +294,7 @@
     "@metamask/multichain-network-controller": "^0.8.0",
     "@metamask/multichain-transactions-controller": "^0.9.0",
     "@metamask/name-controller": "^8.0.3",
-    "@metamask/network-controller": "patch:@metamask/network-controller@npm%3A23.5.0#~/.yarn/patches/@metamask-network-controller-npm-23.5.0-56b87eea93.patch",
+    "@metamask/network-controller": "patch:@metamask/network-controller@npm%3A23.5.1#~/.yarn/patches/@metamask-network-controller-npm-23.5.0-56b87eea93.patch",
```

Comment:
Presumably the patch should be updated here as well? It's saying we're still using a patch of 23.5.0 here.

Reply:
Sure, updated in [13c14e1](https://github.com/MetaMask/metamask-extension/pull/33349/commits/13c14e18690ac710acdf4b23e71ff12916bb79f5).